doc/docs/intro.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 1
+slug: /
```

Comment:
The overview should still be first on the sidebar, right now it displays as the second item.
![image](https://github.com/user-attachments/assets/82987360-7c2f-49e0-9c84-d7dc87f20ed3)